packages/hardhat/hardhat.config.ts

```diff
@@ -21,8 +21,9 @@ const etherscanApiKey = process.env.ETHERSCAN_API_KEY || "DNXJA8RX2Q3VZ4URQIWP7Z
 
 const config: HardhatUserConfig = {
   solidity: {
-    version: "0.8.17",
+    version: "0.8.29",
```

Comment:
I had to update the Solidity version and add viaIR to be able to compile the contracts.